### ISSUE TITLE
Spam "error writing JSON status 200" in volume server logs

### ISSUE THREAD

**Issue by Programmeris** [seaweedfs/seaweedfs#3964]
Sponsors SeaweedFS via Patreon https://www.patreon.com/seaweedfs

**Describe the bug**
I have 24 volume servers in my SeaweedFS cluster. At some point in the logs of each of them began to spam an errors;
` error writing JSON status 200: write tcp 10.233.83.45:8080->172.25.7.32:38652: write: connection reset by peer
`
`
  error writing JSON status 200: write tcp 10.233.80.50:8080->172.25.7.34:47346: write: broken pipe
`

**System Setup**
SeaweedFS 3.32

- 3 master servers
- 3 filer servers
- 24 volume server. One volume server per HDD.
- MongoDB as filer store
- Kubernetes 1.19.2

**Expected behavior**
There are no errors in volume server logs

**Screenshots**
![Снимок экрана от 2022-11-10 16-44-08](https://user-images.githubusercontent.com/49879947/201111967-4bf4ee84-8599-458b-b490-6a3f82981ecd.png)
![Снимок экрана от 2022-11-10 17-07-19](https://user-images.githubusercontent.com/49879947/201113092-f65575fe-3b97-4a62-b515-3a17790d775d.png)

**Additional context**
I have multiple clusters for different environments: dev, qa, prod. The configuration above describes my prod environment. The problem is reproduced only on it.  dev and qa environments have the same cluster topology. The main difference between the environments is the amount of incoming data and therefore the number of volumes on the volume servers. I found a few issues with a bug similar to mine. But they are all closed:

- https://github.com/seaweedfs/seaweedfs/issues/1108
- https://github.com/seaweedfs/seaweedfs/issues/1222

I also found one unclosed issue:

- https://github.com/seaweedfs/seaweedfs/issues/2581

I assume that the problem is related to the vacuum process on thousands volume count

**Comment by chrislusf** [comment]
the error is when writing from volume server to the client 172.25.7.34 and 172.25.7.32. what are these two servers?

**Comment by Programmeris** [comment]
SeaweedFS is deployed on separate nodes with no extraneous load.
172.25.7.32 and 172.25.7.34 are those from these nodes.

**Comment by chrislusf** [comment]
Are they running filers or volume servers?

**Comment by Programmeris** [comment]
Yes. Server volume and filer replicas are deployed on these nodes.

**Comment by Programmeris** [comment]
@chrislusf Are there suggestions?

**Comment by Programmeris** [comment]
> cluster.ps
* filers 3
  * 10.233.78.33:8888 (30GB 3.32 758d70bd)
    signature: 315025073
  * 10.233.73.35:8888 (30GB 3.32 758d70bd)
    signature: 315025073
  * 10.233.82.36:8888 (30GB 3.32 758d70bd)
    signature: 315025073
* volume servers 24
  * data center: DefaultDataCenter
    * rack: rack2
      * seaweedfs-volume-20.seaweedfs-volume:8080 (30GB 3.32 758d70bd)
      * seaweedfs-volume-13.seaweedfs-volume:8080 (30GB 3.32 758d70bd)
      * seaweedfs-volume-0.seaweedfs-volume:8080 (30GB 3.32 758d70bd)
      * seaweedfs-volume-10.seaweedfs-volume:8080 (30GB 3.32 758d70bd)
      * seaweedfs-volume-23.seaweedfs-volume:8080 (30GB 3.32 758d70bd)
      * seaweedfs-volume-8.seaweedfs-volume:8080 (30GB 3.32 758d70bd)
      * seaweedfs-volume-18.seaweedfs-volume:8080 (30GB 3.32 758d70bd)
      * seaweedfs-volume-6.seaweedfs-volume:8080 (30GB 3.32 758d70bd)
    * rack: rack3
      * seaweedfs-volume-2.seaweedfs-volume:8080 (30GB 3.32 758d70bd)
      * seaweedfs-volume-16.seaweedfs-volume:8080 (30GB 3.32 758d70bd)
      * seaweedfs-volume-9.seaweedfs-volume:8080 (30GB 3.32 758d70bd)
      * seaweedfs-volume-17.seaweedfs-volume:8080 (30GB 3.32 758d70bd)
      * seaweedfs-volume-7.seaweedfs-volume:8080 (30GB 3.32 758d70bd)
      * seaweedfs-volume-1.seaweedfs-volume:8080 (30GB 3.32 758d70bd)
      * seaweedfs-volume-11.seaweedfs-volume:8080 (30GB 3.32 758d70bd)
      * seaweedfs-volume-15.seaweedfs-volume:8080 (30GB 3.32 758d70bd)
    * rack: rack1
      * seaweedfs-volume-12.seaweedfs-volume:8080 (30GB 3.32 758d70bd)
      * seaweedfs-volume-22.seaweedfs-volume:8080 (30GB 3.32 758d70bd)
      * seaweedfs-volume-21.seaweedfs-volume:8080 (30GB 3.32 758d70bd)
      * seaweedfs-volume-5.seaweedfs-volume:8080 (30GB 3.32 758d70bd)
      * seaweedfs-volume-14.seaweedfs-volume:8080 (30GB 3.32 758d70bd)
      * seaweedfs-volume-3.seaweedfs-volume:8080 (30GB 3.32 758d70bd)
      * seaweedfs-volume-19.seaweedfs-volume:8080 (30GB 3.32 758d70bd)
      * seaweedfs-volume-4.seaweedfs-volume:8080 (30GB 3.32 758d70bd)

**Comment by chrislusf** [comment]
Added some debug message to the logs.

**Comment by Programmeris** [comment]
Thank you. Sorry for the long answer - a lot of work at the end of the year :) I will plan on the SeaweedFS update in this cluster and after that I will unsubscribe about the result.

**Comment by Programmeris** [comment]
@chrislusf So, I upgraded the production cluster where this problem was observed. In the logs I see the following:

I0221 12:35:49.770454 common.go:106 error writing JSON status /status 200: write tcp 10.233.78.46:8080->172.25.7.33:41044: write: connection reset by peer

172.25.7.33 is a k8s node ip

Maybe error containes in this function https://github.com/seaweedfs/seaweedfs/blob/26dbc6c905189ec471801447e382473ed5b15073/weed/server/volume_server_handlers_admin.go#L31 ?

**Comment by chrislusf** [comment]
the request may come from the liveness or readiness probes.

**Comment by DG-Wangtao** [comment]
I think it is caused by the problem [http probes that do return 200 but hang afterwards do not fail if their size is above 10 Kb #122948](https://github.com/kubernetes/kubernetes/issues/122948) of k8s
